SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -2820,10 +2820,11 @@ bool CFlowOutput::WriteVolume_Output(CConfig *config, unsigned long Iter){
       return true;
     }
   } else {
-    return ((Iter > 0) && Iter % config->GetVolume_Wrt_Freq() == 0);
+    if (config->GetFixed_CL_Mode() && config->GetFinite_Difference_Mode()) return false;
+    return ((Iter > 0) && Iter % config->GetVolume_Wrt_Freq() == 0) || force_writing;
```

Comment:
This isn't the cleanest implementation but is the least intrusive as it is mostly contained in CFlowOutput. @talbring 

The other way to do this (which I have a slight preference towards) would be to add an option to disable volume output completely in COutput itself. I would prefer it because it would be a more general function that could possibly be used in other situations as well. But it would be a pretty powerful option that might cause conflicts with other options/solvers if not handled with care.

Unfortunately, the way it is now, there is one state variable (Finite_Difference_Mode) that is stored in the config @pcarruscag . But everything else is contained in the solver

Reply:
I am ok with that!